<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JsonToolClass</class>
 <widget class="QWidget" name="JsonToolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>JsonTool</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>缩进</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spin_indentation">
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="maximum">
          <number>20</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_format">
         <property name="text">
          <string>格式化</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_rmformat">
         <property name="text">
          <string>去除格式</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_save">
         <property name="toolTip">
          <string>将右侧文本框的内容保存到文件</string>
         </property>
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_find">
         <property name="placeholderText">
          <string>在此输入要查找的内容</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_next">
         <property name="text">
          <string>下一个</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_prev">
         <property name="text">
          <string>上一个</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QPlainTextEdit" name="te_dst">
       <property name="placeholderText">
        <string>此处显示处理后的文本:-)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPlainTextEdit" name="te_src">
       <property name="placeholderText">
        <string>在此输入需要处理的原文本……</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="json_tool.qrc"/>
 </resources>
 <connections/>
</ui>
